pytorch3d/renderer/mesh/rasterizer.py

```diff
@@ -110,7 +110,9 @@ def __init__(self, cameras=None, raster_settings=None) -> None:
 
     def to(self, device):
         # Manually move to device cameras as it is not a subclass of nn.Module
-        self.cameras = self.cameras.to(device)
+        cameras = self.cameras
+        if cameras is not None:
```

Comment:
This looks like a significant bug you are fixing, which is separate from this change. Thank you. It would be good to put it in its own PR which would include a test for this problem. 

(Or let us know and we can do it internally.)